src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -177,6 +177,7 @@ trait ScalaSettings extends StandardScalaSettings with Warnings {
    * -Y "Private" settings
    */
   val Yhelp           = BooleanSetting    ("-Y", "Print a synopsis of private options.")
+  val Ylint           = BooleanSetting    ("-Ylint", "Make -Xlint more annoying.")
```

Comment:
Not sure about this part...

Reply:
Me neither.

Reply:
`-Ylint-a-little-when-you-can-lint-a-lot`